apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -231,6 +231,9 @@ export function flyoutCategory(workspace, functionEditorOpen = false) {
     fields: {
       NAME: Blockly.Msg.PROCEDURES_DEFNORETURN_PROCEDURE,
     },
+    extraState: {
+      userCreated: true,
+    },
```

Comment:
When functions are created outside of the editor, they still need this default value applied.